config/locales/forms/2023/lettings/tenancy_information.en.yml

```diff
@@ -0,0 +1,66 @@
+en:
+  forms:
+    2023:
+      lettings:
+        tenancy_information:
+          joint:
+            page_header: ""
+            check_answer_label: "Is this a joint tenancy?"
+            hint_text: ""
+            question_text: "Is this a joint tenancy?"
+
+          startertenancy:
+            page_header: ""
+            check_answer_label: "Is this a starter or introductory tenancy?"
+            hint_text: "This is also known as an ‘introductory period’."
+            question_text: "Is this a starter tenancy?"
+
+          tenancy:
+            tenancy_type:
+              page_header: ""
+              tenancy:
+                check_answer_label: "Type of main tenancy"
+                hint_text: ""
+                question_text: "What is the type of tenancy?"
+              tenancyother:
+                check_answer_label: ""
+                hint_text: ""
+                question_text: "Please state the tenancy type"
+            starter_tenancy_type:
+              page_header: ""
+              tenancy:
+                check_answer_label: "Type of main tenancy after the starter period has ended"
+                hint_text: "This is also known as an ‘introductory period’."
+                question_text: "What is the type of tenancy after the starter period has ended?"
+              tenancyother:
+                check_answer_label: ""
+                hint_text: ""
+                question_text: "Please state the tenancy type"
```

Comment:
Same here, tenancy type and starter tenancy type are two separate pages, might be better as separate keys without "tenancy" as a parent? Does it make sense to keep the page as the parent key (directly after tenancy_information) and any variations underneath that?

Reply:
I think similarly to above I grouped by the question id